pkgs/by-name/pr/proton-pass/linux.nix

```diff
@@ -0,0 +1,61 @@
+{
+  stdenvNoCC,
+  dpkg,
+  makeWrapper,
+  electron,
+  asar,
+  lib,
+  version,
+  fetchurl,
+  pname,
+  meta,
+}:
+
+stdenvNoCC.mkDerivation {
```

Comment:
Not sure if this would work with the `callPackage`, but the reason why this was written with a lambda function was to make it easier for people to potentially override the derivation (`stdenvNoCC.mkDerivation (finalAttrs: { ... })`). Is there a reason why you're not using the lambda here?